go/vt/topo/srv_keyspace.go

```diff
@@ -142,6 +548,75 @@ func (ts *Server) DeleteSrvKeyspace(ctx context.Context, cell, keyspace string)
 	return conn.Delete(ctx, nodePath, nil)
 }
 
+// GetSrvKeyspaceAllCells returns the SrvKeyspace for all cells
+func (ts *Server) GetSrvKeyspaceAllCells(ctx context.Context, keyspace string) ([]*topodatapb.SrvKeyspace, error) {
+	cells, err := ts.GetCellInfoNames(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	srvKeyspaces := make([]*topodatapb.SrvKeyspace, len(cells))
+	for _, cell := range cells {
+		srvKeyspace, err := ts.GetSrvKeyspace(ctx, cell, keyspace)
+		switch {
+		case err == nil:
+			srvKeyspaces = append(srvKeyspaces, srvKeyspace)
+		case IsErrType(err, NoNode):
+			// NOOP
+		default:
+			return srvKeyspaces, err
+		}
+	}
+	return srvKeyspaces, nil
+}
+
+// IsShardQueryServiceDisabled returns served types for given shard across all cells
+func (ts *Server) IsShardQueryServiceDisabled(ctx context.Context, si *ShardInfo) (queryServiceDisabled bool, err error) {
```

Comment:
I don't see any place where `queryServiceDisabled` is being set. Can you recheck this function?
We should also see why no tests failed if this is indeed a bug.

Reply:
Oh this was a function I thought I was going to need, but never end up using it. I just deleted it. There were not tests for this as it was not being used.